src/test/kotlin/ratpack/example/kotlin/SiteTest.kt

```diff
@@ -1,67 +1,54 @@
 package ratpack.example.kotlin
 
-import com.google.common.io.CharStreams
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
-import ratpack.test.MainClassApplicationUnderTest
-import java.io.InputStreamReader
-import kotlin.test.assertEquals
-import kotlin.test.fail
 
 @RunWith(JUnit4::class)
 class SiteTest {
+    val app = kappFromServer(createServer ())
 
-  val aut = MainClassApplicationUnderTest(Main::class.java)
```

Comment:
I would still use `MainClassApplicationUnderTest ` for testing.  It has functionality like Impositions that you will need for "real" testing.